unittests/test_utility.py

```diff
@@ -541,598 +594,611 @@ def test_repr_default():
-        assert 3 == len(l)
-        assert 2 in l
-        assert l == [1, 2, 2]
-        assert l == util.SequenceView([1, 2, 2])
-        assert list(reversed(l)) == [2, 2, 1]
-        assert 1 == l.index(2)
-        assert 2 == l.count(2)
-        assert str(l) == str([1, 2, 2])
-
-        # Assert immutability
-        m = l + [3, 4]
-        assert [1, 2, 2, 3, 4] == m
-        assert isinstance(m, util.SequenceView)
-
-        m = l
-        l += [3, 4]
-        assert m is not l
-        assert [1, 2, 2] == m
-        assert [1, 2, 2, 3, 4] == l
-        assert isinstance(l, util.SequenceView)
-
-        with pytest.raises(TypeError):
-            l[1] = 3
-
-        with pytest.raises(TypeError):
-            l[1:2] = [3]
-
-        with pytest.raises(TypeError):
-            l *= 3
-
-        with pytest.raises(TypeError):
-            del l[:1]
-
-        with pytest.raises(AttributeError):
-            l.append(3)
-
-        with pytest.raises(AttributeError):
-            l.clear()
-
-        with pytest.raises(AttributeError):
-            s = l.copy()
-
-        with pytest.raises(AttributeError):
-            l.extend([3, 4])
-
-        with pytest.raises(AttributeError):
-            l.insert(1, 4)
-
-        with pytest.raises(AttributeError):
-            l.pop()
-
-        with pytest.raises(AttributeError):
-            l.remove(2)
-
-        with pytest.raises(AttributeError):
-            l.reverse()
-
-    def test_mapping(self):
-        d = util.MappingView({'a': 1, 'b': 2})
-        assert 1 == d['a']
-        assert 2 == len(d)
-        assert {'a': 1, 'b': 2} == dict(d)
-        assert 'b' in d
-        assert {'a', 'b'} == set(d.keys())
-        assert {1, 2} == set(d.values())
-        assert {('a', 1), ('b', 2)} == set(d.items())
-        assert 2 == d.get('b')
-        assert 3 == d.get('c', 3)
-        assert {'a': 1, 'b': 2} == d
-        assert d == util.MappingView({'b': 2, 'a': 1})
-        assert str(d) == str({'a': 1, 'b': 2})
-        assert {'a': 1, 'b': 2, 'c': 3} != d
-
-        # Assert immutability
-        with pytest.raises(TypeError):
-            d['c'] = 3
-
-        with pytest.raises(TypeError):
-            del d['b']
-
-        with pytest.raises(AttributeError):
-            d.pop('a')
-
-        with pytest.raises(AttributeError):
-            d.popitem()
-
-        with pytest.raises(AttributeError):
-            d.clear()
-
-        with pytest.raises(AttributeError):
-            d.update({'a': 4, 'b': 5})
-
-        with pytest.raises(AttributeError):
-            d.setdefault('c', 3)
 
+    with pytest.raises(AttributeError):
+        d.pop('a')
+
+    with pytest.raises(AttributeError):
+        d.popitem()
+
+    with pytest.raises(AttributeError):
+        d.clear()
+
+    with pytest.raises(AttributeError):
+        d.update({'a': 4, 'b': 5})
+
+    with pytest.raises(AttributeError):
+        d.setdefault('c', 3)
+
+
+@pytest.fixture
+def random_seed():
+    random.seed(1)
```

Comment:
I don't think we get something out of this fixture. Just do `random.seed(1)` in each of the tests that need it.